TripKitUI/helper/AnnotationViewBuilder.swift

```diff
@@ -62,9 +63,11 @@ public class AnnotationViewBuilder: NSObject {
     return self
   }
   
-  @objc public func build() -> MKAnnotationView? {
-    if let vehicle = annotation as? Vehicle {
-      return build(for: vehicle)
+  @objc open func build(_ usingMarkerAnnotationIfPossible: Bool = false) -> MKAnnotationView? {
```

Comment:
Shouldn't use `_` for a boolean like this, as you otherwise can't tell what the bool means when reading the calling code and it just says `build(true)`.